modules/ROOT/pages/dw-core-functions-map.adoc

```diff
@@ -56,43 +61,25 @@ This example creates an object with the array values, using the index (plus 1) a
 %dw 2.0
 output application/json
 ---
-{
-  (["joe", "pete", "matt"] map (item, index) -> {(index + 1): item})
-}
+['a', 'b', 'c'] map ((value, index) -> (index + 1) ++ '_' ++ value)
 ----
 
 ==== Output
 
 [source,JSON,linenums]
 ----
-{
-  "1": "joe",
-  "2": "pete",
-  "3": "matt"
-}
+[ "1_a", "2_b", "3_c" ]
 ----
 
-
 === Example
 
 If the parameters of the `mapper` function are not named, the index can be
-referenced with `&#36;&#36;`, and the value with `&#36;`.
-So the previous example can be written like this.
-
-==== Source
-
-[source,DataWeave, linenums]
-----
-%dw 2.0
-output application/json
----
-["joe", "pete", "matt"] map (($$ + 1) ++ ":" ++ upper($))
-----
-
-
-=== Example
-
-This example shows how to turn an array of key-value pairs into an object.
+referenced with `&#36;&#36;`, and the value with `&#36;`. This example
+iterates over each item in the input array `['joe', 'pete', 'matt']`
+and returns an array of objects where the index is selected as the key.
```

Comment:
I suggest: ...with the index selected as the key. (This gets around the passive.)

Reply:
I'm trying to make it clear that we're using $$ to select the index and use that as a key.